manager.go

```diff
@@ -276,14 +276,6 @@ func (m *Manager) CheckUpdates(ctx context.Context) time.Duration {
 		interval = minimalCheckInterval
 	}
 
-	m.cfgMu.RLock()
```

Comment:
is the reason locking is not needed anymore is that is happening before we enter the loop where the config could change?